src/app/components/main-action/main-action.component.html

```diff
@@ -3,6 +3,8 @@
         <button
             *ngSwitchCase="actionTypes.button"
             mat-stroked-button
+            [id]="action.id"
+            [disabled]="action.disabled"
```

Comment:
why do we need these additional params?

Reply:
can we add unit test to cover this scenario?

Reply:
Added test for disabled button